src/clib/pio_lists.c

```diff
@@ -251,22 +254,21 @@ int pio_add_to_iodesc_list(io_desc_t *iodesc)
 /** Get an iodesc.
  *
  * @param ioid ID of iodesc to get.
- * @internal
+ * @returns pointer to the iodesc struc.
  */
 io_desc_t *pio_get_iodesc_from_id(int ioid)
 {
     io_desc_t *ciodesc = NULL;
 
-    if(current_iodesc != NULL && current_iodesc->ioid == abs(ioid))
+    if (current_iodesc != NULL && current_iodesc->ioid == abs(ioid))
         ciodesc = current_iodesc;
-    for(ciodesc = pio_iodesc_list; ciodesc != NULL; ciodesc = ciodesc->next)
-    {
-        if(ciodesc->ioid == abs(ioid))
-        {
-            current_iodesc = ciodesc;
-            break;
-        }
-    }
+    else
```

Comment:
Cleaning as I touch the code. We did our whitespace sweep - any other whtiespace issues should be addressed as found. Eventually they will decrease to a smaller and smaller number.